modules/raft/src/main/java/org/apache/ignite/internal/raft/server/impl/JraftServerImpl.java

```diff
@@ -264,7 +268,13 @@ public CompletableFuture<Void> startAsync(ComponentContext componentContext) {
             opts.setSnapshotTimer(JRaftUtils.createTimer(opts, "JRaft-SnapshotTimer"));
         }
 
-        requestExecutor = JRaftUtils.createRequestExecutor(opts);
+        requestExecutor = new ThreadPoolExecutor(
```

Comment:
It seems to be the same as `Executors.newFixedThreadPool(opts.getRaftRpcThreadPoolSize(), ... factory ...)